test-aether-integration-module/conf/pip/requirements.txt

```diff
@@ -8,10 +8,8 @@
 # then run 'docker-compose -f docker-compose-test.yml run integration-test pip_freeze'	#
 #                                                                              		#
 #########################################################################################
-aet.consumer==0.9.1
-aether.client==0.0.0
-aether.mocker==0.0.0
-aether.saladbar==0.0.0
+aet.consumer>=0.9.1
+aether.client>=0.9.0
```

Comment:
I think that you change this manually this you try this in the `primary-requirements.txt

```
######################################################################################################
#                                                                                                    #
#                             Primary requirements                                                   #
#                                                                                                    #
#                                 CHANGE ME !!!                                                      #
#                                                                                                    #
# Update me and then run 'docker-compose -f docker-compose-test.yml run integration-test pip_freeze' #
#                                                                                                    #
######################################################################################################


aet.consumer
aether.client=`cat VERSION`
aether.mocker=`cat VERSION`
aether.saladbar ##  ehem ehem

coverage
flake8
kafka
mock
pytest
pytest-runner
requests
spavro
```

Reply:
Obviously, tweaking a little bit the `pip_freeze` command :nerd_face: